languages/go/exercises/concept/using-map/usingmap.go

```diff
@@ -0,0 +1,18 @@
+package usingmap
+
+// var priceTable
+
+// TransactionLog represent a single item in a transaction
+type TransactionLog struct {
+	itemName string
+	month    string
+	qty      int
+}
+
+func calculateTotalIncome(monthlyIncome map[string]int) int {
+	panic("Not Implemented")
+}
+
+func calculateMonthlyIncome(transactionLogs []TransactionLog) (map[string]int, error) {
```

Comment:
same here: exported function and comment

Reply:
Suggestion (needs a rename from `TransactionLog` to `Transaction`):
```suggestion
// CalculateMonthly generates an overview of the income per month.
func CalculateMonthly(transactions []Transaction) (map[string]int, error) {
```